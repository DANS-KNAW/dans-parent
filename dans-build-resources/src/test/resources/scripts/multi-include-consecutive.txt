A series of includes:

#include <single-line.txt>
#include <multi-line.txt>

An some more text.

