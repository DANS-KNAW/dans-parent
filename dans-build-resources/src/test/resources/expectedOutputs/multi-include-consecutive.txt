A series of includes:

line 1
line 1
line 2
line 3


An some more text.

